browser/brave_browser_main_extra_parts.cc

```diff
@@ -6,9 +6,11 @@
 #include "brave/browser/brave_browser_main_extra_parts.h"
 
 #include "brave/browser/brave_browser_process_impl.h"
+#include "brave/browser/net/brave_system_request_handler.h"
```

Comment:
In the future, I think, the whole thing (potentially including `UrlContext`) should live in `/net`. @bridiver WDYT?

Reply:
this one needs to stay in `brave/browser` because its deps would create a layering violation (like the one currently in safe_browsing), but https://github.com/brave/brave-core/pull/4268/files#diff-ce86d8f829640e5e88ce2b71c4e22b01R13 will move to `brave/services/network`